examples/app-with-extension-point/README.md

```diff
@@ -0,0 +1,31 @@
+# Grafana App Plugin which adds a plugin UI extension point
+
+This example demonstrates how to add a plugin extension point that can be extended by other plugins. It works by bundling three different app plugins within the same bundle. First we have the `myorg-extensionpoint-app` that creates an extension point. Then we have the `src/plugins/myorg-a-app` and `src/plugins/myorg-b-app` that extends the `myorg-extensionpoint-app` UI with a link and a command.
+
+> The way we have nested app plugins in this example is not recommended to use in production.
+
+## Guides in this example
+
+| **Example**                                                      | **Source**       |
+| ---------------------------------------------------------------- | ---------------- |
+| [How to add an extension point?](#how-to-add-an-extension-point) | [App.tsx#L12-18] |
+
+### How to add an extension point?
+
+**Example:** [App.tsx#L12-18]
```

Comment:
**Nit:** should we link the source? Or is this intended to be a foot-note?

Reply:
I have no idea. Please help me here 🙈